contrib/screen-13-macros/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+#![warn(missing_docs)]
+
+// Proc-macros and doc-tests currently have some limitations, so we allow referncing self via
+// screen_13:
+// https://github.com/rust-lang/cargo/issues/9886
+// https://github.com/bkchr/proc-macro-crate/issues/10
+#[allow(unused_extern_crates)]
+extern crate self as screen_13_macros;
+
+pub mod vertex;
+pub use screen_13::driver::ash;
```

Comment:
This may not need to be `pub`

Reply:
I added a comment in https://github.com/attackgoat/screen-13/pull/56/commits/19ebea2e278fce9dec9d22eccb7d6e0ab039c59f explaining why ash is re-exported. Basically I wanted to contain the workarounds required to make proc macros work in edge-cases like doc tests to the contrib macro crate only.